spec.md

```diff
@@ -228,10 +228,40 @@ how platforms might expose these values to their users.
 | metadata | JSON object | An opaque object of metadata for a service plan. Controller treats this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing brokers and controllers for fields that aid in the display of catalog data. |
 | free | boolean | When false, service instances of this plan have a cost. The default is true. |
 | bindable | boolean | Specifies whether service instances of the service plan can be bound to applications. This field is OPTIONAL. If specified, this takes precedence over the `bindable` attribute of the service. If not specified, the default is derived from the service. |
+| [schemas](#schema-object) | object | Schema definitions for service instances and bindings for the plan. |
 
 
 \* Fields with an asterisk are REQUIRED.
 
+##### Schema Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [service_instance](#service-instances-object) | object | The schema definitions for creating and updating a service instance. |
+| [service_binding](#service-bindings-object) | object | The schema definition for creating a service binding. Used only if the service plan is bindable. |
+
+
+##### Service Instances Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definition for creating a service instance. |
+| [update](#input-parameters-object) | object | The schema definition for updating a service instance. |
+
+
+##### Service Bindings Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| [create](#input-parameters-object) | object | The schema definition for creating a service binding. |
+
+
+##### Input Parameters Object
+
+| Response field | Type | Description |
+| --- | --- | --- |
+| parameters | JSON schema object | The schema definition for the input parameters. Platforms MUST support at least [JSON Schema draft v4](http://json-schema.org/), but SHOULD be prepared to support multiple versions. Broker authors MUST provide the version of JSON Schema they are using. Schemas MUST NOT contain any external references and MUST be no larger than 64kB. Each input parameter is expressed as a property within a JSON object. |
```

Comment:
I'm ok with the requirements of this field. I'm just wondering whether it would read better if everything after the first sentence were moved to a non-tabular format. Meaning, put it in a paragraph/section right after this table so that we have more space to be more explicit. For example:

+++
While the presence of schema in the Plan is OPTIONAL, if it is present then the following applies:
- Platforms MUST support at least [JSON Schema draft v4](http://json-schema.org).
- Platforms SHOULD be prepared to support other versions of JSON Schema well.
- The `$schema` keyword MUST be present in the schema with the version of the JSON Schema being used.
- Schemas MUST NOT contain any external references (e.g. use the `$ref` keyword).
- Schemas MUST NOT be larger than 64kB.
- Each input parameter MUST be expressed as a child of the `properties` field in the schema.

+++

Just allows us to be a little more verbose and show the exact json entities we're talking about.

Reply:
Did you want to mention that `type` is required? Or is that obvious?

Reply:
We're leaving that up to the spec, since this changes in v6 schema.
Moving the text out of the table - check back in 2 mins!